bucket/siyuan-note.json

```diff
@@ -0,0 +1,47 @@
+{
+    "homepage": "https://github.com/siyuan-note/siyuan",

```

Comment:
Hey could you follow the usual order when it comes to Scoop packages?

Look at accesser for example

https://github.com/ScoopInstaller/Extras/blob/master/bucket/accesser.json

Reply:
Thanks for the good check!
But the my original fork has been deleted, I fix the issue in a new PR.
[Pull Request #7254 · ScoopInstaller/Extras](https://github.com/ScoopInstaller/Extras/pull/7254)